### DISCUSSION TITLE
Multiple storage class for same filesystem?

### DISCUSSION THREAD

**Discussion by aplufr** [rook/rook#12420] [Ideas]
Hello,

We have a ceph fs cluster deployed through rook / helm chart, that has also deployed the storage class for accessing data.
Everything is working fine but we would like to have two storageclass, one with the reclaim policy set to delete and the other one to retain.

Currently our work around is either to manually change the PV upon creation to update the reclaim policy or duplicate the storage class created by helm to change that value.

We do not want to create a second pool, filesystem and so on to handle that use case. 

Thanks.

**Comment by Madhu-1** [comment]
Thats is not a workaround i would say, You can create a new storageclass with same content but with a different name, Rook creates only one storageclass for you. IMHO it doesn't make sense to create duplicate copies of storageclass from Rook helm chart.
